cob_hardware_config/cob4-2/config/base_joint_limits.yaml

```diff
@@ -1,31 +0,0 @@
-joint_limits:
```

Comment:
> we still could set the joint_limits in the yaml in case we want another limit for the wheel joints...

yes

> overwrite the values in the base.urdf.xacro

I guess: no
The URDF defines the hardware properties.